contents/handbook/people/compensation.mdx

```diff
@@ -111,7 +111,9 @@ Your manager is responsible for monitoring and specifically reviewing your perfo
 
 ## Severance
 
-If your time at PostHog comes to an end, we will pay severance to support your transition into your next role. If PostHog decides to end your contract after the first 3 months of employment have been completed, we will give you 4 months' notice and pay you for those months. If we agree with you that a shorter notice period is more appropriate, we will still pay you a total of 4 months of salary (even if you leave immediately). In either case, it is likely we will ask you to stop working immediately. 
+We believe that average performance gets a generous severance.
```

Comment:
No need to say we believe. Average performance gets generous severance at PostHog.